src/library/scala/collection/mutable/AnyRefMap.scala

```diff
@@ -293,8 +293,8 @@ class AnyRefMap[K <: AnyRef, V] private[collection] (defaultEntry: K => V, initi
     }
   }
 
-  /** Adds a new key/value pair to this map and returns the map. */
-  def +=(key: K, value: V): this.type = { update(key, value); this }
+  @compileTimeError("use += with a tuple argument instead")
+  def +=(key: K, value: V): this.type = ???
```

Comment:
Should we lose this method?  The replacement requires a tuple allocation.

Reply:
You can migrate to `addOne(K, V)` if you want to avoid that.

Reply:
Makes me thing of BigDecimal "add" vs Int +, but ok.  Good to know.  Thanks.